noggin/controller/user.py

```diff
@@ -271,7 +284,7 @@ def user_settings_otp_delete(ipa, username):
         username = session.get('noggin_username')
         token = form.token.data
         try:
-            ipa.otptoken_del(ipatokenuniqueid=token)
+            ipa.otptoken_del(a_ipatokenuniqueid=token)['result']
```

Comment:
Seems weird to access the `result` key when we're not storing it in a variable.

Reply:
Yeah, that can go away.